internal/view/actions.go

```diff
@@ -107,12 +109,12 @@ func pluginActions(r Runner, aa ui.KeyActions) {
 		}
 		aa[key] = ui.NewKeyAction(
 			plugin.Description,
-			execCmd(r, plugin.Command, plugin.Background, plugin.Args...),
+			execCmd(r, plugin.Command, plugin.Background, plugin.ConfirmDialog, plugin.Args...),
 			true)
 	}
 }
 
-func execCmd(r Runner, bin string, bg bool, args ...string) ui.ActionHandler {
+func execCmd(r Runner, bin string, bg bool, confirmDialog bool, args ...string) ui.ActionHandler {
```

Comment:
New signature should be something like func pluginAction(r Runner, p config.Plugin) ui.ActionHandler {}

Reply:
This could also enable us to add input dialogs as well at some point...